src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -61,12 +71,69 @@ EffectManifestPointer PitchShiftEffect::getManifest() {
     pitch->setNeutralPointOnScale(0.0);
     pitch->setRange(-1.0, 0.0, 1.0);
 
+    EffectManifestParameterPointer rangeMode = pManifest->addParameter();
+    rangeMode->setId("rangeMode");
+    rangeMode->setName(QObject::tr("Mode"));
+    rangeMode->setShortName(QObject::tr("Mode"));
+    rangeMode->setDescription(QObject::tr(
+            "Set the range mode. There are three possible options "
+            "based on the knob position:\n"
+            "neutral:\n"
+            "\t - knob: default\n"
+            "\t - range: from the lowest pitch to the highest pitch\n"
+            "positive:\n"
+            "\t - knob: maximum\n"
+            "\t - range: from the default pitch to the highest pitch\n"
+            "negative:\n"
+            "\t - knob: minimum\n"
+            "\t - range: from the default pitch to the lowest pitch"));
+    rangeMode->setValueScaler(EffectManifestParameter::ValueScaler::Linear);
+    rangeMode->setDefaultLinkType(EffectManifestParameter::LinkType::Linked);
+    rangeMode->setNeutralPointOnScale(0.0);
+    rangeMode->setRange(-1.0, 0.0, 1.0);
+
+    EffectManifestParameterPointer semitonesMode = pManifest->addParameter();
+    semitonesMode->setId("semitonesMode");
+    semitonesMode->setName(QObject::tr("Semitones"));
+    semitonesMode->setShortName(QObject::tr("Semitones"));
+    semitonesMode->setDescription(QObject::tr(
+            "Set the pitch scale to the semitones mode "
+            "instead of the contiguous mode (set as default)."));
```

Comment:
I don't think these explanations are very intuitive. I have some proposals which seem more user friendly. wdyt?
```suggestion
            "Change pitch in semitone-steps instead of smoothly));
```

Reply:
Agree, looks good. Thank you.